HTML/EN/cmdwrappers

```diff
@@ -295,7 +295,11 @@ END %]
 [% END %]
 
 [% BLOCK albuminfo %]
-	<a [% PROCESS albumItemHRef %] target="browser" class="browseItemLink">[% item.text | html %]</a>
+	[% IF item.albumTitle %]
```

Comment:
Why would you use the `albumTitle` to decide whether we can provide a link? We'd need `item.albumId || item.itemobj.album.id`.

Reply:
A choice based on what the trackinfo block above does. But yes, checking for the actual parameter that the url requires makes much more sense.